frontend/hub/common/ExternalLink.tsx

```diff
@@ -0,0 +1,51 @@
+import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
+import { ReactNode } from 'react';
+
+interface IProps {
+  children: ReactNode;
+  'data-cy'?: string;
+  href: string;
+  variant?: 'default' | 'download' | 'menu' | 'nav';
+}
+
+// variants:
+// download - no external link icon (role download)
+// menu - top nav question mark menu (Customer Support, Training)
+// nav - left side nav (Documentation, Terms of Use)
+
+export const ExternalLink = ({
+  children,
+  'data-cy': dataCy,
+  href,
+  variant = 'default',
+}: IProps) => {
+  if (!href || !children) {
+    return null;
+  }
+
+  const iconStyle = {
```

Comment:
What about usage of styled component? That is more closer to what we want to use. I think this PR should use them, because we want to be consistent.

https://styled-components.com/

Reply:
Will do a follow-up pr for the styled component.